tailwind.config.ts

```diff
@@ -178,5 +179,15 @@ export default {
       ...breakpoints(theme('screens')),
     }),
   },
-  plugins: [],
+  plugins: [
+    plugin(({ addUtilities }) => {
+      addUtilities({
+        '.transformBox-content': { 'transform-box': 'content-box' },
+        '.transformBox-border': { 'transform-box': 'border-box' },
+        '.transformBox-fill': { 'transform-box': 'fill-box' },
+        '.transformBox-stroke': { 'transform-box': 'stroke-box' },
+        '.transformBox-view': { 'transform-box': 'view-box' },
+      });
+    }),
+  ],
```

Comment:
This setting may cause issues with tailwind merge when using `cn(...)`. Don't think this case is likely to happen, so just flagging.

Reply:
add an entry in twmerge